docs/install.md

```diff
@@ -0,0 +1,98 @@
+Installation Guide
+==================
+
+=== "Install with go"
```

Comment:
Don't forget that dib also depends on `graphviz` package to generate .dot files for graph rendering